### DISCUSSION TITLE
New badge

### DISCUSSION THREAD

**Discussion by prayogaea** [community/community#18256] [Other Feature Feedback, Questions, & Ideas]
Hopefully in the future github will continue to add various badges for open source contributors around the world. In Indonesia, they are very enthusiastic about the Badges that Github gives to open source contributors.

**Comment by Sulagna-Dutta-Roy** [comment]
All the best !!!

**Comment by JagMolar** [comment]
Until today, github is being a creative environment and open to all developers. Presumably they continue to promote our activity to enrich the project.
